rskj-core/src/test/java/co/rsk/core/WalletFactory.java

```diff
@@ -20,14 +20,13 @@
 
 import co.rsk.config.TestSystemProperties;
 import org.ethereum.datasource.HashMapDB;
-import org.ethereum.datasource.KeyValueDataSource;
-import org.ethereum.datasource.LevelDbDataSource;
+import org.ethereum.datasource.RocksDbDataSource;
 
 public class WalletFactory {
 
     public static Wallet createPersistentWallet(String storeName) {
```

Comment:
it looks like this method is not used and can be deleted

Reply:
Done